app/controllers/uploads_controller.rb

```diff
@@ -301,11 +303,17 @@ def create_multipart
     file_size = params.require(:file_size).to_i
     upload_type = params.require(:upload_type)
 
-    if file_size_too_big?(file_name, file_size)
-      return render_json_error(
-        I18n.t("upload.attachments.too_large_humanized", max_size: ActiveSupport::NumberHelper.number_to_human_size(SiteSetting.max_attachment_size_kb.kilobytes)),
-        status: 422
-      )
+    if upload_type == "backup"
```

Comment:
This is some of the jankyness mentioned in the PR description that I hope to address in a subsequent PR.